mrp_partial_production/models/stock.py

```diff
@@ -28,10 +29,14 @@ def _split(self, qty, restrict_partner_id=False):
                 'package_id': aml.package_id.id or False,
                 'owner_id': aml.owner_id.id or False,
             }
+            available_quantity += product_uom_qty
             aml_vals.append(vals)
         new_move_id = super(StockMove, self)._split(qty, restrict_partner_id)
         for vals in aml_vals:
             vals['move_id'] = new_move_id
             self.env['stock.move.line'].create(vals)
-        self.browse(new_move_id)._recompute_state()
+        new_move = self.browse(new_move_id)
+        new_move._update_reserved_quantity(
+            new_move.product_uom_qty, available_quantity, new_move.location_id, strict=False)
```

Comment:
Are you sure it should be strict=False?

Reply:
yes, we got it from after debugging to following line
https://github.com/odoo/odoo/blob/11.0/addons/stock/models/stock_move.py#L950